whois-query/src/test/java/net/ripe/db/whois/query/integration/AbuseCTestIntegration.java

```diff
@@ -45,10 +45,21 @@
             "mnt-by:        TEST-MNT\n" +
             "source:        TEST",
 
+            "person:        AbuseC Person\n" +
+            "address:       VENUSNIC, see http://www.venusnic.net\n" +
+            "nic-hdl:       ABUSEC-PERSON-TEST\n" +
+            "abuse-mailbox: abuseperson@ripe.net\n" +
+            "mnt-by:        TEST-MNT\n" +
+            "source:        TEST",
+
             "organisation:  ORG-TEST-1\n" +
             "abuse-c:       ABU-TEST\n" +
             "source:        TEST",
 
+            "organisation:  ORG-TEST-ABUSEC-PERSON\n" +
+            "abuse-c:       ABUSEC-PERSON-TEST\n" +
```

Comment:
This can't happen, only a role can be used as an abuse-c reference, not a person. Can you change the test data to use a role instead?